### PR TITLE
Implement SMA STP 10.0 SE Hybrid Inverter

### PR DESCRIPTION
**Pull request by tsicking**
Implementation of the SMA STP 10.0 SE Hybrid Inverter including an SMA battery and a DC charger.

### CODE REVIEW

File: io.openems.edge.ess.sma/src/io/openems/edge/ess/sma/stpxx3se/battery/Config.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	@AttributeDefinition(name = "Is enabled?", description = "Is this Component enabled?")
+	boolean enabled() default true;
+
+	@AttributeDefinition(name = "Modbus-ID", description = "ID of Modbus bridge.")
+	String modbus_id() default "modbus0";
+
+	@AttributeDefinition(name = "Modbus-Unit-ID", description = "Modbus Unit-ID.")
+	int modbusUnitId() default 3;
+	
+	@AttributeDefinition(name = "Charge Max Voltage", description = "Maximum voltage for charging")
```
Comment (clehne) [inline io.openems.edge.ess.sma/src/io/openems/edge/ess/sma/stpxx3se/battery/Config.java]: please add unit to description :  Maximum voltage for charging in V. 
Same for dischargeMinVoltage

File: io.openems.edge.ess.sma/src/io/openems/edge/ess/sma/stpxx3se/battery/SmaBatteryImpl.java [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+import io.openems.edge.common.component.OpenemsComponent;
+import io.openems.edge.common.startstop.StartStop;
+import io.openems.edge.common.startstop.StartStoppable;
+import io.openems.edge.common.taskmanager.Priority;
+import io.openems.edge.ess.sma.enums.SetControlMode;
+import io.openems.edge.ess.sma.sunnyisland.EssSmaSunnyIsland;
+
+@Designate(ocd = Config.class, factory = true)
+@Component(//
+		name = "Ess.Sma.StpSe.Battery", //
```
Comment (clehne) [inline io.openems.edge.ess.sma/src/io/openems/edge/ess/sma/stpxx3se/battery/SmaBatteryImpl.java]: Sunny Island factory ID is  "Ess.SMA.SunnyIsland"
So we should probably use  something like "Ess.SMA.STP-SE.Battery" here.
Also in the ESS and in the DC Charger itself

**Review by clehne — CHANGES_REQUESTED** [review]
Minor open points. Please see comments.

File: io.openems.edge.ess.sma/src/io/openems/edge/ess/sma/stpxx3se/battery/Config.java (hunk 1 above)
Comment (tsicking) [reply io.openems.edge.ess.sma/src/io/openems/edge/ess/sma/stpxx3se/battery/Config.java]: Okay, hab es geändert.

File: io.openems.edge.ess.sma/src/io/openems/edge/ess/sma/stpxx3se/battery/SmaBatteryImpl.java (hunk 2 above)
Comment (tsicking) [reply io.openems.edge.ess.sma/src/io/openems/edge/ess/sma/stpxx3se/battery/SmaBatteryImpl.java]: Ich war erst nicht sicher, ob das "-" in der Factory-ID ein Problem ist, scheint aber nicht so zu sein.

**Review by clehne — APPROVED** [review]

**Pull request merged**